packages/neuron-ui/src/services/remote/apiMethodWrapper.ts

```diff
@@ -21,14 +21,7 @@ export const RemoteNotLoadError = {
   },
 }
 
-export const controllerNotLoaded = (controllerName: string) => ({
-  status: 0 as 0,
-  message: {
-    content: `${controllerName} controller not loaded`,
-  },
-})
-
-export const controllerMethodWrapper = (controllerName: string) => (
+export const apiMethodWrapper = () => (
```

Comment:
The most outside wrapper is redundant now.

Reply:
Updated w/ 18e7a327.